File: FeedAPI/FeedAPI/FeedAPI/Models/Article.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace FeedAPI.Models
{
    public class Article
    {        
        public string Title { get; set; }
        public string Author { get; set; }
        public string Source { get; set; }
        public string Link { get; set; }
        public string ImageLink { get; set; }
        public string Content { get; set; }
        public DateTime PublishDate { get; set; }
    }
}

File: UI/Client/src/app/Models/Article.ts
export class Article{
    Title: string;
    Author: string;
    Source: string;
    Link: string;
    ImageLink: string;
    Content: string;
    PublishDate: string;

    constructor(_title: string, _author: string){
        this.Title = _title;
        this.Author = _author;
        this.Source = _author;
        this.Link = _author;
        this.ImageLink = _author;
        this.Content = _author;
        this.PublishDate = _author;
    }
}